packages/create-cloudflare/src/helpers/cli.ts

```diff
@@ -27,6 +27,9 @@ export const C3_DEFAULTS = {
 	git: true,
 	open: true,
 	ts: true,
+	// TODO: Come up with a better value for this
+	template:
+		"https://github.com/cloudflare/workers-sdk/tree/main/packages/create-cloudflare/templates/hello-world",
```

Comment:
Could use degit shortcut?

```suggestion
		"cloudflare/workers-sdk#main/packages/create-cloudflare/templates/hello-world",
```